src/utils/media-utils.js

```diff
@@ -22,14 +22,38 @@ export const resolveMedia = async url => {
   return resolved;
 };
 
+export const upload = file => {
+  const formData = new FormData();
+  formData.append("media", file);
+  return fetch(mediaAPIEndpoint, {
+    method: "POST",
+    body: formData
+
+    // We do NOT specify a Content-Type header like so
```

Comment:
Though I didn't know this off hand, its just documentation for how `FormData` works with the `fetch` API, seems a bit excessive. I would have looked it up if I was confused.